library/worldgen/src/testmod/resources/quilt.mod.json

```diff
@@ -2,10 +2,10 @@
   "schema_version": 1,
   "quilt_loader": {
     "group": "org.quiltmc.qsl",
-    "id": "quilt_data_testmod",
```

Comment:
This is probably out-of-scope for this PR; Just do a direct commit to both branches and all will be fine